cli/src/pcluster/templates/cdk_builder_utils.py

```diff
@@ -54,6 +54,15 @@
 PCLUSTER_LAMBDA_PREFIX = "pcluster-"
 
 
+def _get_target_group_name(cluster_name, pool_name):
```

Comment:
- We can have some minimal unit tests for this function.
- This function seems to be relevant for the login nodes stack. We can keep it in the login nodes stack instead of having it in the `cdk_builder_utils` file which is used for reuseable code.
    - [nice-to-have] Alternatively make it generic e.g. the function accepts a list of strings and a pattern used to combine the strings to fit a desired length requirement.

Reply:
Done!